### PR TITLE
fix use RequestStack instead of Request

### PR DESCRIPTION
**Pull request by upskaling**
I tried to fix issues #65

Let me know if I can improve Pull requests

- https://github.com/Harborn-digital/cookie-consent-bundle/issues/65

### CODE REVIEW

**Comment by matthijsch** [comment]
Thanks! Can you fix the unit test?

**Comment by upskaling** [comment]
@matthijsch If there is anything else you want me to correct, don't hesitate.

File: Cookie/CookieChecker.php [hunk 1]
```diff
@@ -20,9 +21,9 @@ class CookieChecker
      */
     private $request;
 
-    public function __construct(Request $request)
+    public function __construct(RequestStack $request)
     {
-        $this->request = $request;
+        $this->request = $request->getCurrentRequest();
```
Comment (jdreesen) [inline Cookie/CookieChecker.php:26]: You shouldn't call `getCurrentRequest()` in the constructor, but in the method in which it is used. The reason is, that the constructor is called during service generation (when it is injected into another service) but the actual usage may be later, so the "current request" may be another one at that point.

https://symfony.com/doc/current/service_container/request.html

**Comment by upskaling** [comment]
Thanks for taking the time to look at my code 
I've made the changes, please let me know if there's anything else.

**Comment by matthijsch** [comment]
Sadly we seem to be experiencing some issues with Github and PR's were automatically closed and can't be reopened. Thank you for your work, I've created a new PR #125 and also replaced RequestStack for the logger

**Pull request closed without merging**